doc/workflow.md

```diff
@@ -0,0 +1,66 @@
+# Workflow
+
+#### **kube-spawn-dir**
+`/var/lib/kube-spawn` by default, can be set with `--dir`
+
+#### **cluster-dir**
+`<kube-spawn-dir>/default` by default, can be set with `--cluster-name`
```

Comment:
Why is the title of the section `cluster-dir`? Is it a flag name? If so, it should be `--cluster-dir`. If not, it should be `Cluster name`.

Ditto for `kube-spawn-dir`, it could be `kube-spawn base directory`.